Author: Alastair McKinstry <mckinstry@debian.org>
Description: Force .mo files to all be little-endian so that the libnewt package can be M-A: same
Last-Updated: 2013-05-14
Forwarded: no

Index: newt-0.52.15/po/Makefile
===================================================================
--- newt-0.52.15.orig/po/Makefile	2013-03-25 14:16:36.000000000 +0000
+++ newt-0.52.15/po/Makefile	2013-05-14 18:41:36.000000000 +0100
@@ -41,7 +41,7 @@
 report:
 	@for cat in $(CATALOGS); do \
 		echo -n "$$cat: "; \
-		msgfmt -v --statistics -o /dev/null $$cat; \
+		msgfmt --endianness little -v --statistics -o /dev/null $$cat; \
 	done
 
 clean:
